src/elemSection/page.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+.inputcontainer{
+  display:flex;
+  width: 300px;
+  flex-wrap:wrap;
+  gap:10px;
+  justify-content: start;
+  margin-left:20px;
+}
+.createcontainer,
+.updatecontainer{
+  display:flex;
+  gap:10px;
+}
+.input_color{
+  height:20px;
+  width:30px;
+}
+.input_text{
+  width:150px;
+  height:20px;
+}
+.btncontainer{
+  margin-left:20px;
+  display:flex;
+  gap:10px;
+  margin-top:10px;
+  width:300px;
+  justify-content: start;
+}
+.board{
+  margin-top:20px;
+  height:50vh;
+  width:99vw;
+  background-color: black;
+  margin-left:0
+
+}
+.row{
+  display:flex;
+  height:80px;
+
+}
+.controls{
+  width:110px;
+  display:flex;
+  flex-wrap:wrap;
+  gap:5px;
+
+}
+.name{
+text-align:center;
+ width:300px;
+ overflow-x: visible;
+height:20px;
+  color:white;
+  white-space: nowrap;
+}
+.track{
+  position:relative;
+  width:calc(99vw - 110px);
+  border-bottom:1px dashed white;
+}
+
+.car{
```

Comment:
all css should not be in one file, css of each component should be close to this component